resources/views/locations/bulk-delete.blade.php

```diff
@@ -0,0 +1,70 @@
+@extends('layouts/default')
```

Comment:
I *bet* - if you wanted to - you could make this generic and re-usable across a couple of different "Object Types™". But let's cross that bridge when we come to it.

Reply:
I'm not sure what you mean there, but we can discuss further offline I think.